Apps/WebClient/src/ClientApp/src/components/navmenu/navHeader.vue

```diff
@@ -113,9 +113,7 @@ export default class HeaderComponent extends Vue {
     }
 
     private mounted() {
-        if (this.oidcIsAuthenticated) {
-            this.loadOidcUser();
-        }
+        this.onPropertyChanged();
```

Comment:
this looks odd. Do you know it does the same?

Reply:
I checked - onPropertyChanged() consists of the same code, so this should be fine.

Reply:
Yes that was raised as duplicated since both functions did the same thing, i renamed the method so hopefully it makes more sense.